### ISSUE TITLE
cf dev start fails with: ssh: handshake failed: 

### ISSUE THREAD

**Issue by kamleshg** [vmware-archive/pcfdev#286]
Full error:
`kgokal@~ $ cf dev start
Using existing image.
Starting VM...
FAILED
Error: failed to start VM: ssh connection timed out: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain.
`
This is on OSX, with VirtualBox 5.2.6 (latest)

**Comment by kamleshg** [comment]
I saw the several tickets mentioned VM's within VM's but this is just VirtualBox appliance so not related right?

**Comment by sclevine** [comment]
Sorry for the extended delay. Can you try with Virtualbox 5.1?

**Comment by ykhadilkar** [comment]
Same issue here. Tried on both 5.1.36 and 5.2.10 same error. pcfdev was working until I upgraded Virtual Box. 
`
Error: failed to start VM: ssh connection timed out: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain.`

**Comment by mhilmen** [comment]
Looks like I found away to get pcf dev back up and running on my Mac.

I had tried the latest versions of VB 4.3 (which actually failed to install on my Mac), 5.0, 5.1, and 5.2 getting failures on the Starting VM stage each time:

"FAILED
Error: failed to start VM: ssh connection timed out: dial tcp 192.168.11.11:22: connect: host is down."

or 

"FAILED
Error: failed to start VM: ssh connection timed out: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain."

I used all permutations of "cf dev stop", and "cf dev destroy" (I also tried removing the pcf dev VM manually via  the VirtualBox UI) all without success.

I wasn't until I uninstalled pcf dev with this command "cf uninstall-plugin pcfdev" and re-installed it that I got the VM to come up successfully again.  I am using VirtualBox v 5.2.12.

I used the following site to learn how to completely remove cf dev
https://docs.pivotal.io/pcf-dev/install-osx.html
